skiko/build.gradle.kts

```diff
@@ -154,6 +156,33 @@ kotlin {
         withJava()
     }
 
+    val nativeTarget = when (target) {
+        "macos-x64" -> macosX64()
+        // "linux-x64" -> linuxX64()
+        // "windows-x64" -> mingwX64()
+        else -> null
+    }
+    nativeTarget?.apply {
+        compilations.getByName("main") {
```

Comment:
Does native builds by default now? Let's put it under flag for now.

Reply:
I.e. `./gradlew -Pskiko.native.enabled=true` and `if (project.properties.get("skiko.native.enabled") == "true")` protecting native tasks.

Reply:
done that